### PR TITLE
Added Object 28: Device RF Capabilities

### PR DESCRIPTION
**Pull request by asoloway64**
new pull request template

### CODE REVIEW

**Comment by seanmcilroy29** [comment]
This PR is being submitted directly against PROD.
A feature branch should be create from dev and the PR pushed against the newly created branch

File: 28.xml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  https://www.omaspecworks.org/about/intellectual-property-rights/
+
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+	<Object ObjectType="MODefinition">
+		<Name>Device</Name>
+		<Description1><![CDATA[This is a device management object that should be used for capturing the RF capabilities of the device.]]></Description1>
+		<ObjectID>28</ObjectID>
+		<ObjectURN>urn:oma:lwm2m:oma:28:1.0</ObjectURN>
```
Comment (seanmcilroy29) [inline 28.xml]: urn should be `urn:oma:lwm2m:oma:28`

File: 28.xml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+  The above license is used as a license under copyright only.  Please
+  reference the OMA IPR Policy for patent licensing terms:
+  https://www.omaspecworks.org/about/intellectual-property-rights/
+
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+	<Object ObjectType="MODefinition">
+		<Name>Device</Name>
```
Comment (seanmcilroy29) [inline 28.xml]: In the DDF file, the Name is `Device RF Capabilities`

**Comment by asoloway64** [comment]
I fixed both the object name and the urn.

**Comment by jpradocueva** [comment]
This PR cannot be merged against the "Production" branch.

**Comment by asoloway64** [comment]
Opened #609 to merge to dev branch.

**Pull request closed without merging**